src/Radicle/Internal/PrimFns.hs

```diff
@@ -709,8 +709,10 @@ purePrimFns = fromList $ allDocs $
           let allMod = fromEnv e
           toImport <- case v_ of
                 Just vs -> do
-                  is <- Protolude.toList <$> traverse isAtom vs ?? toLangError (OtherError "import: must be given a vector of symbols to import")
-                  pure $ Map.restrictKeys allMod (Set.fromList is)
+                  is <- Set.fromList <$> Protolude.toList <$> traverse isAtom vs ?? toLangError (OtherError "import: must be given a vector of symbols to import")
+                  if Set.null (is Set.\\ Map.keysSet allMod)
```

Comment:
No idea if this `if ... then ... else` is idiomatic here, it's been a while since I've last written any Haskell.

Reply:
Yup - looks good.